app.js

```diff
@@ -0,0 +1,85 @@
+
+
+var P1score = 0;
+var CPUscore = 0;
+var round = 1;
+
+while (round <= 5) {
+
+  alert("Round: " + round);
+  console.log(round);
+
+  var P1move = prompt("Choose one: Rock Paper Scissors");
+  console.log(P1move);
+
+  var CPUmove = Math.random();
```

Comment:
Rounding this down to an integer and multiplying it by the highest number you want to possibly get would make it a lot cleaner and easier to deal with.